core-web/libs/portlets/edit-ema/ui/src/lib/dot-seo-image-preview/dot-seo-image-preview.component.scss

```diff
@@ -23,7 +23,7 @@ $image-height: 23.875rem;
 :host ::ng-deep {
     .dot-seo-image-preview__default {
         .pi {
-            font-size: $font-size-xxxl;
+            font-size: $font-size-xxxxl;
```

Comment:
It seems that the `$font-size-xxxxl` is meant for icons. We should create a separate variable for them, thoughts, @dotCMS/frontend?

Reply:
If we use it more than once, yes but with a identificable name.

Reply:
Just added `$icon-xl` on `_common.scss` and remove this `$font-size-xxxxl`